### PR TITLE
OCPBUGS-24404: add ignore for SAST scan

### PR DESCRIPTION
**Pull request by tkashem**

### CODE REVIEW

File: .synk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# References:
+# https://docs.snyk.io/scan-applications/snyk-code/using-snyk-code-from-the-cli/excluding-directories-and-files-from-the-snyk-code-cli-test
+# https://docs.snyk.io/snyk-cli/commands/ignore
+exclude:
+  global:
+    - vendor/**
+    - "**/*_test.go"
+    - staging/src/k8s.io/api/testdata/**
+    - hack/testdata/**
```
Comment (dgrisonnet) [inline .synk:9]:
```suggestion
    - **/testdata/**
```

**Review by dgrisonnet — APPROVED** [review]
/lgtm

**Comment by soltysh** [comment]
/remove-label backports/unvalidated-commits
/label backports/validated-commits

/override ci/prow/unit
this is being tracked elsewhere

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by rphillips** [comment]
cancelling hold since the bot added it

/hold cancel

**Pull request merged**